### PR TITLE
Add python bindings option

### PR DESCRIPTION
**Pull request by ahoarau**
This just adds a python binding option that defaults to *ON* to be safe.
This is in part due to https://github.com/ros-visualization/rviz/issues/1382

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -143,7 +143,10 @@ find_package(catkin REQUIRED
   visualization_msgs
 )
 
-find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
+option(BUILD_PYTHON_BINDINGS "Build the python bindings using sip" ON)
```
Comment (rhaschke) [inline CMakeLists.txt]: `sip` is only one option to build the python bindings.
```suggestion
option(BUILD_PYTHON_BINDINGS "Build python bindings" ON)
```

**Comment by ahoarau** [comment]
I'm using the c++ `librviz` for one of my projet. I had the exact same error as #1382 that stopped me from compiling anything on melodic.  One way to pass the option : 
```
catkin config --cmake-args -DBUILD_PYTHON_BINDINGS=OFF
```
But it applies for all the packages in the ws, it's true. Better than nothing :) 
Copying the sources in the project that needs `librviz` and adding it with `add_subdirectory(rviz)` might work just as well and allow to pass the option to it individually.

**Comment by ahoarau** [comment]
So are there any issues on this one ?

**Comment by rhaschke** [comment]
The issue #1382 should be solved via https://github.com/ros-visualization/python_qt_binding/pull/64.
Could you please verify? If so, I suggest to cancel this PR.

> `catkin config --cmake-args -DBUILD_PYTHON_BINDINGS=OFF`

The option name is too generic. Better use something like `RVIZ_PYTHON_BINDINGS`.

**Comment by rhaschke** [comment]
Closing due to inactivity. Issue should be fixed as indicated in https://github.com/ros-visualization/rviz/pull/1383#issuecomment-525206015

**Pull request closed without merging**